### PR TITLE
added a finish process to the choose pin status flow

### PR DESCRIPTION
**Pull request by mikecann**
This is as a result of discussions on this issue: https://github.com/Haskkor/react-native-pincode/issues/13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "dev": "yarn build -watch",
```
Comment (mikecann) [inline package.json]: Oh sorry I left this line in, feel free to remove if you want

**Comment by mikecann** [comment]
Okay finally got my build working again and tested that this works as expected

**Comment by Haskkor** [comment]
Tested and working, sorry for the delay :).

**Comment by mikecann** [comment]
No worries at all, thanks for your hard work.

**Pull request merged**